src/app/query/bitcoin/transaction/use-check-utxos.ts

```diff
@@ -36,6 +37,44 @@ export function filterOutIntentionalUtxoSpend({
   });
 }
 
+interface CheckInscribedUtxosByBestinslotArgs {
+  inputs: btc.TransactionInput[];
+  txids: string[];
+  client: ReturnType<typeof useBitcoinClient>;
```

Comment:
Would it be a better API to pass just the individual methods we need?

Reply:
hm, yeah, it would make code more explicit